apollo-audit/apollo-audit-api/src/main/java/com/ctrip/framework/apollo/audit/api/ApolloAuditLogRecordApi.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2023 Apollo Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package com.ctrip.framework.apollo.audit.api;
+
+import com.ctrip.framework.apollo.audit.annotation.OpType;
+import java.util.List;
+import java.util.Map;
+
+public interface ApolloAuditLogRecordApi {
+
+  Map extractSpan();
```

Comment:
context?

```suggestion
  ApolloAuditSpanContext extractSpanContext();
```

Reply:
ApolloAuditSpanContext is at impl package, while ApolloAuditLogRecordApi is at api package. Either, extractSpan is to making context to obj which could be transport.